.github/workflows/push.yml

```
@@ -252,10 +252,12 @@ jobs:
             echo "IMAGES='$IMAGES'" >> "$GITHUB_OUTPUT"
           fi
 
-      - name: Prepare docker deps
-        id: docker-deps
-        run: |
-          ./scripts/ci/10_prepare-docker-deps.sh
+      - name: Prepare node-modules for docker
+        uses: ./.github/actions/docker-deps
```

actionlint (1.7.4)

description is required in metadata of "Prepare yarn install docker layer" action at "/home/jailuser/git/.github/actions/docker-deps/action.yml"

(action)